### PR TITLE
[zh-cn]: update the translation of HTTP `Allow` header

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/http/headers/allow/index.md [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
 ---
 
 {{HTTPSidebar}}
 
-**`Allow`** 首部字段用于枚举资源所支持的 HTTP 方法的集合。
+**`Allow`** 标头列出了资源支持的方法集。
 
-若服务器返回状态码 {{HTTPStatus("405")}} `Method Not Allowed`，则该首部字段亦需要同时返回给客户端。如果 `Allow` 首部字段的值为空，说明资源不接受使用任何 HTTP 方法的请求。这是可能的，比如服务器需要临时禁止对资源的任何访问。
+当服务器响应带有 {{HTTPStatus("405")}} `Method Not Allowed` 状态码时必须发送此标头，以表示可以使用哪些请求方法。`Allow` 空标头表示该资源不允许任何请求方法，例如，这种情况可能针对某个特定资源会临时出现。
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/allow/index.md:12]:
```suggestion
当服务器响应带有 {{HTTPStatus("405")}} `Method Not Allowed` 状态码时必须发送此标头，以表示可以使用哪些请求方法。`Allow` 空标头表示该资源不允许请求任何方法，例如，这种情况可能针对某个特定资源会临时出现。
```

**Review by jasonren0403 — APPROVED** [review]
Sorry for late review. LGTM 👍 
Edit: maybe you can consider this https://github.com/mdn/translated-content/pull/20120#discussion_r1595318020

File: files/zh-cn/web/http/headers/allow/index.md (hunk 1 above)
Comment (jasonren0403) [reply files/zh-cn/web/http/headers/allow/index.md]: 或者“不允许使用任何请求方法”？

**Comment by yin1999** [comment]
> Sorry for late review. LGTM 👍 Edit: maybe you can consider this [#20120 (comment)](https://github.com/mdn/translated-content/pull/20120#discussion_r1595318020)

I would also like this idea :+1:

File: files/zh-cn/web/http/headers/allow/index.md [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
 ---
 
 {{HTTPSidebar}}
 
-**`Allow`** 首部字段用于枚举资源所支持的 HTTP 方法的集合。
+**`Allow`** 标头列出了资源支持的方法集。
 
-若服务器返回状态码 {{HTTPStatus("405")}} `Method Not Allowed`，则该首部字段亦需要同时返回给客户端。如果 `Allow` 首部字段的值为空，说明资源不接受使用任何 HTTP 方法的请求。这是可能的，比如服务器需要临时禁止对资源的任何访问。
+当服务器响应带有 {{HTTPStatus("405")}} `Method Not Allowed` 状态码时必须发送此标头，以表示可以使用哪些请求方法。`Allow` 空标头表示该资源不允许请求任何方法，例如，这种情况可能针对某个特定资源会临时出现。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/allow/index.md:12]:
```suggestion
当服务器响应带有 {{HTTPStatus("405")}} `Method Not Allowed` 状态码时必须发送此标头，以表示可以使用哪些请求方法。`Allow` 空标头表示该资源不允许使用任何请求方法，例如，这种情况可能针对某个特定资源会临时出现。
```

**Pull request merged**